### DISCUSSION TITLE
BTRFS snapshots on GRUB for Debian/Ubuntu-based distros

### DISCUSSION THREAD

**Discussion by KorraHarraway** [Antynea/grub-btrfs#145] [Q&A]
Hi,

I would like to know if by executing the makefile script  (or moving manually the files) is it possible to install the feature into the GRUB (of course, for btrfs installations with subvolumes already properly set in FSTAB) of latest Debian and Ubuntu-based  (20.04+) distros.

If possible anyway, how should I proceed?

Tks in advance :)

**Comment by Kr1ss-XD** [comment]
On Ubuntu, you can follow the instructions for manual installation from the [README](https://github.com/Antynea/grub-btrfs/blob/master/README.md#manual), (via `make`).

This _should_ work for Debain too, although I'm not absolutely sure if there are any dependencies which are on too old versions in the repos.

Once the script is installed, configure it to your needs via `/etc/default/grub-btrfs/config`, and run `update-grub`. If you want automatic GRUB updates after new snapshots are made, run `systemctl enable --now grub-btrfs.path`. Depending on your snapshot location, you'll need to edit this unit file beforehand.

That's documented in the README though. Wish you success, _cheers_ !

**Reply by KorraHarraway** [reply]
Tested and it is working here :)

**Reply by Kr1ss-XD** [reply]
Great, thanks for the feedback !
